references/sdk.md

```diff
@@ -504,15 +504,15 @@ login(self, username, password)
 Login to a remote server. 
 ## update_compute_plan
 ```python
-update_compute_plan(self, compute_plan_id: str, data: Union[dict, substra.sdk.schemas.UpdateComputePlanSpec], auto_batching: bool = True, batch_size: int = 20) -> substra.sdk.models.ComputePlan
+update_compute_plan(self, key: str, data: Union[dict, substra.sdk.schemas.UpdateComputePlanSpec], auto_batching: bool = True, batch_size: int = 20) -> substra.sdk.models.ComputePlan
 ```
 
 Update compute plan.
 As specified in the data dict structure, output trunk models of composite
 traintuples cannot be made public.
 
 **Arguments:**
- - `compute_plan_id (str, required)`: Id of the compute plan
+ - `key (str, required)`: Id of the compute plan
```

Comment:
Could you update the arg description as well so that there is no more mention of "ID".